### PR TITLE
MaxPool2d in the torch

### PR DESCRIPTION
**Pull request by z-a-f**
Stack:
&nbsp;&nbsp;&nbsp;&nbsp;:black_circle:&nbsp; **#22765 MaxPool2d in the torch**&nbsp;&nbsp;[:green_heart:](https://our.internmc.facebook.com/intern/diff/D16102608/)

the pooling signature is the same as the non-quantized one. Adding it to the native_functions.yaml

Differential Revision: [D16102608](https://our.internmc.facebook.com/intern/diff/D16102608/)

### CODE REVIEW

File: torch/nn/modules/pooling.py [hunk 1]
```diff
@@ -70,10 +75,6 @@ def forward(self, input):
                             self.padding, self.dilation, self.ceil_mode,
                             self.return_indices)
 
-    def extra_repr(self):
```
Comment (jerryzh168) [inline torch/nn/modules/pooling.py]: why do we remove this?

File: torch/nn/quantized/modules/pooling.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from __future__ import absolute_import
```
Comment (jerryzh168) [inline torch/nn/quantized/modules/pooling.py]: can we do this in quantized/modules/__init__.py?

**Review by jerryzh168 — APPROVED** [review]
lgtm, with minor comments.

File: torch/nn/quantized/modules/pooling.py (hunk 2 above)
Comment (zafartahirov) [reply torch/nn/quantized/modules/pooling.py]: I am still fighting the Python community about that. The current hack is to use it, but to be careful with `from ... import *`

File: torch/nn/modules/pooling.py (hunk 1 above)
Comment (zafartahirov) [reply torch/nn/modules/pooling.py]: This is a duplicate from the parent -- it is inherited

File: torch/nn/quantized/modules/pooling.py (hunk 2 above)
Comment (jerryzh168) [reply torch/nn/quantized/modules/pooling.py]: oh, I mean do we need this file? we can just create a alias in `quantized/modules/__init__.py`

Comment (zafartahirov) [reply torch/nn/quantized/modules/pooling.py]: This is to mirror non-quantized version. Technically we don't need it, but creating an alias will prevent from `from torch.nn.quantized.pooling import MaxPool2d`

Comment (jerryzh168) [reply torch/nn/quantized/modules/pooling.py]: yeah I mean we can put this in init? and i think people will do `from torch.nn.quantized import MaxPool2d`

Comment (zafartahirov) [reply torch/nn/quantized/modules/pooling.py]: Yes. I think right now the users can still do `from torch.nn.quantized import MaxPool2d`. Do you think I should move the alias to init, and not create a new file for it?

Comment (zafartahirov) [reply torch/nn/quantized/modules/pooling.py]: removed the pooling -- now only __init__ has it

**Comment by gchanan** [comment]
@zafartahirov please provide a better title for the PR.  It is not possible to really have any idea what this PR does based on the title.

**Pull request closed without merging**